app/src/buildconfig.ts

```diff
@@ -288,12 +246,12 @@ function get_bugsnag_key(): string | false {
   return bugsnag_key;
 }
 
-function get_conductor_url(): string {
-  const url = import.meta.env.VITE_CONDUCTOR_URL;
-  if (url) {
-    return url;
+function get_conductor_urls(): string[] {
+  const urls = import.meta.env.VITE_CONDUCTOR_URL;
+  if (urls) {
+    return urls.split(' ');
   } else {
-    return 'http://localhost:8154';
+    return ['http://localhost:8154'];
```

Comment:
Would be good to log that this fall through has occurred and put this as a constant somewhere in the build config rather than hardcoding here.

Reply:
Logging added.